www/src/templates/template-docs-markdown.js

```diff
@@ -68,10 +68,12 @@ class DocsTemplate extends React.Component {
         <Helmet>
           <title>{page.frontmatter.title}</title>
           <meta name="description" content={page.excerpt} />
-          <meta name="og:description" content={page.excerpt} />
+          <meta property="og:description" content={page.excerpt} />
+          <meta property="og:title" content={page.frontmatter.title} />
+          <meta property="og:type" content="article" />
+          <meta property="og:image" content={`../../logo.svg`} />
```

Comment:
Same here.. ? I'm not 100% this will be picked up, either.

Reply:
I supposed that this relative path will be resolved in accordance with project structure. Based on file where I've made this change..